### ISSUE TITLE
Cursor for Menu, Select, Combobox should reset to first item when using keyboard and no item selected

### ISSUE THREAD

**Issue by mannycarrera4** [Workday/canvas-kit#2781]
## 🐛 Bug Report

When a user uses the keyboard to navigating to a Menu, Select or Combobox, opens the popup, and uses the arrow keys to navigating, if they hit ESCAPE without select an item, when the popup opens back up, the cursor should reset to the FIRST item in the list.

Currently, the cursor stay on the last focusable element instead of reseting to the first item in the list.

I think we just need to check if there's a selected item in the model and if not, reset the cursor id to the first item in the list.

FWIW this isn't a bug. Accessibility has mentioned that either functionality is OK, but maybe reseting the cursor might provide a better experience.

**Comment by myvuuu** [comment]
This should be an update to the collection system.
